test.py

```diff
@@ -1,4 +1,4 @@
 from test1 import func
```

Comment:
This import is unused. Remove it to avoid clutter.

```suggestion
Remove the import statement.
```